### DISCUSSION TITLE
Email change

### DISCUSSION THREAD

**Discussion by fabian57fabian** [ultralytics/hub#385] [Q&A] [answered]
Is it possible to change the account email?
I've created my account using google login, and used a wrong address.
Can i change it?

**Comment by kalenmike** [comment, marked as answer]
@fabian57fabian We not have any way for you to change your email address at the moment. The only way we can achieve this at the moment would be for you to deal with us directly.

1. Create a new HUB account with the correct email.
2. Send us an email from your current account and one from the new account asking for them to be merged. web@ultralytics.com
3. We will then merge the two accounts and delete the first one.

**Reply by fabian57fabian** [reply]
Thank you.
